CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 [...]
 
+- **[FIX]** `Profile` reacquires its hovering effect (similar to other clickable items)
```

Comment:
`gets back` intead of `reacquires`

Reply:
`intead` > `instead` 😬

Reply:
meta typo

Reply:
No one agrees on how to write in english x)